const.go

```diff
@@ -43,28 +43,29 @@ const (
 	urlRemoveProfPic = "accounts/remove_profile_picture/"
 	urlFeedSaved     = "feed/saved/"
 	urlSetBiography  = "accounts/set_biography/"
-	urlEditProfile   = "accounts/edit_profile"
 	urlFeedLiked     = "feed/liked/"
+	urlPendingFollowers = "friendships/pending?"
```

Comment:
For example `friendships/pending` is currently used by https://github.com/ahmdrz/goinsta/blob/master/const.go#L62